### ISSUE TITLE
Just recently ran SVN update, now program doesnt work

### ISSUE THREAD

**Issue by outlawimmortal** [kolton/d2bot-with-kolbot#1323]
I just ran an SVN update for KOLBOT and it said there was a conflict on trunk/d2bs/D2BS.dll

everytime i try to run anything i get an error saying its either not designed to run on windows or contains an error. (game.exe - Bad Image)  and it runs one instance of diablo, but then i try to run another it says i can only run one diablo at a time! what happend?! i dont want to reinstall for i have spent hours putting my profiles together!

and theres a (!) on the file and even on my kolbot folder, what did i do wrong? all i did was run an svn update

**Comment by outlawimmortal** [comment]
so i hit "solve" and it got rid of the (!) still had the issue, but then i "reverted" the file and i guess it worked! for now i have no issues.

**Comment by DarkHorseDre** [comment]
revert, as in youre now running the old version of the file?

**Comment by outlawimmortal** [comment]
i believe so? i right clicked on the file and saw an option to "revert" it. I was trying to see if there was a way i could go back to the previous version, and this seemed to solve my problem. i havent tried anything other than just the loaders with map... but i havent seen any issues yet

**Comment by noah-** [comment]
you probably had a merge conflict, pick "theirs" and complete the merge

**Comment by outlawimmortal** [comment]
im sorry im not too sure what you mean but picking "theirs" would i have to run another svn update? because right now there is no conflict since i reverted

**Comment by DarkHorseDre** [comment]
yeah hes saying when you run the update and there's a conflict, you should be able to choose "theirs" (the new version) or 'Mine' (your current file).  Pick the newest one (if you didnt modify the file yourself)

**Comment by outlawimmortal** [comment]
okay! so im good reverting it like i did? just next time i run an update with a conflict just do what you said to do? thank you guys very much for your time! Very much appreciated

**Comment by DarkHorseDre** [comment]
no it sounds as if you are running the old version. if I'm right you need to update all of your files as described. running an old version of a file with newer files could cause other issues

**Comment by outlawimmortal** [comment]
ok so i ran another svn update and it didnt do any kind of updates, it said it was at revision 18307 and it didnt change anything. 

do i need to update just the file itself?

**Comment by DarkHorseDre** [comment]
Sounds like all files are updated - I find Tortoise a bit finicky so not always sure whats going on!
you can compare your file with the current files:  https://github.com/kolton/d2bot-with-kolbot

The date of the current file should be the same as the time you updated, which would also imply it is up to date

*the .old files are the older versions which have been replaced during the update
